docs/data/material/getting-started/templates/templates.md

```diff
@@ -21,10 +21,12 @@ Sections of each layout are defined either by comments or use of separate files,
 These templates can be combined with one of the [example projects](/material-ui/getting-started/example-projects/) to form a complete starter app.
 
 :::info
```

Comment:
I don't love the back-to-back callouts here. Maybe we should move the Toolpad callout up higher? It could work after line 15: "...from the source code or via CodeSandbox or StackBlitz."